lib/matplotlib/axis.py

```diff
@@ -2074,10 +2070,12 @@ def set_ticks_position(self, position):
             self.set_tick_params(which='both', top=True, labeltop=True,
                                  bottom=False, labelbottom=False)
             self._tick_position = 'top'
+            self.offsetText.set_va('bottom')
```

Comment:
I prefer not using the abbreviations for better readability
```suggestion
            self.offsetText.set_verticalalignment('bottom')
```